Add a "What Next" section to the Getting Started Guide

deveshks opened the pull request:
<!---
Thank you for your soon to be pull request. Before you submit this, please
double check to make sure that you've added a news file fragment. In pip we
generate our NEWS.rst from multiple news fragment files, and all pull requests
require either a news file fragment or a marker to indicate they don't require
one.

To read more about adding a news file fragment for your PR, please check out
our documentation at: https://pip.pypa.io/en/latest/development/contributing/#news-entries
-->
This PR refers to issue: https://github.com/pypa/pip/issues/7879

deveshks commented:
Can someone please take a look at this PR, also I am not sure why the check of continuous-integration/travis-ci is still stuck.

pradyunsg commented on docs/html/development/getting-started.rst:
```diff
@@ -123,8 +123,22 @@ To build it locally, run:
 The built documentation can be found in the ``docs/build`` folder.
 
+What Next?
+===============
```
The heading-underline should have the same number of characters as the heading.

```suggestion
==========
```

deveshks replied:
Hi @pradyunsg,

I have pushed a new commit to do that. Once the PR is approved, I will squash both commits in one and merge the PR

deveshks commented:
Hi @pradyunsg ,

I have addressed the comment you had in the PR, but the check of continuous-integration/travis-ci is stuck again, while all others have passed

deveshks commented:
Hi @pradyunsg ,

So I just closed and reopened the PR like you did earlier, and now all checks have passed. Could you take a look at this PR?

uranusjr commented on docs/html/development/getting-started.rst:
```diff
@@ -123,8 +123,22 @@ To build it locally, run:
+The following pages may be helpful for new contributors on where to look next
+in order to start contributing.
+
+* Some `good first issues`_ on Github for new contributors
```
```suggestion
* Some `good first issues`_ on GitHub for new contributors
```

deveshks replied:
Thanks for the comment @uranusjr. I fixed the casing and squashed all commits into one. Please take a look, and if it looks good, feel free to approve and merge the PR

uranusjr commented:
Sorry @pradyunsg we had a race condition and I accidentally reset your review approval 😞

deveshks commented:
Hi @uranusjr,

I accidentally re-requested a review from you, apologies.

pradyunsg commented:
All is well as long as the PR doesn't accidentally get closed/merged when not intended. :)

pradyunsg commented:
Just waiting on the CI checks now.

deveshks commented:
`All is well as long as the PR doesn't accidentally get closed/merged when not intended. :)`

That's very true @pradyunsg . I also see that all checks have passed now, you can merge the PR :)

pradyunsg commented:
Thanks @deveshks! ^>^
